### PR TITLE
Fix #6154: Enable Solana Dapps by Default

### LINKED ISSUE #6154: Enable Solana Dapps support by default
### Description: 

Enable Solana Dapps support by default

### PR DESCRIPTION
**Pull request by StephenHeaps**
## Security Review
https://github.com/brave/security/issues/1129

## Summary of Changes
- Removes the `isSolanaDappsEnabled` feature flag

This pull request fixes: 
- #6154
- #6653

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New or updated UI has been tested across:
  - [x] Light & dark mode
  - [x] Different size classes (iPhone, landscape, iPad)
  - [x] Different dynamic type sizes

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: Client/Frontend/Browser/UserScriptManager.swift [hunk 1]
```diff
@@ -199,8 +199,7 @@ class UserScriptManager {
       }
       
       // Inject SolanaWeb3Script.js
-      if WalletDebugFlags.isSolanaDappsEnabled,
-         tab.isPrivate == false,
+      if tab.isPrivate == false,
          Preferences.Wallet.WalletType(rawValue: Preferences.Wallet.defaultSolWallet.value) == .brave {
         let script = ScriptLoader.loadUserScript(named: "SolanaWeb3Script") ?? tab.walletSolProviderScripts[.solanaWeb3]
```
Comment (Brandon-T) [inline Client/Frontend/Browser/UserScriptManager.swift]: Can we remove this so it does not default to the brave core one which is insecure :)

If the script can't be loaded, don't inject anything.

**Review by Brandon-T — CHANGES_REQUESTED** [review]

File: Client/Frontend/Browser/UserScriptManager.swift (hunk 1 above)
Comment (StephenHeaps) [reply Client/Frontend/Browser/UserScriptManager.swift]: Yup, I can remove the fallback here 👍

Comment (StephenHeaps) [reply Client/Frontend/Browser/UserScriptManager.swift]: Fixed in [8d7d695](https://github.com/brave/brave-ios/pull/6530/commits/8d7d695d90b85469dd5aa6feba8ffc7b583cd373)

**Review by kdenhartog — APPROVED** [review]
LGTM for security

**Review by nuo-xu — APPROVED** [review]

File: Client/Frontend/Browser/UserScriptManager.swift (hunk 1 above)
Comment (StephenHeaps) [reply Client/Frontend/Browser/UserScriptManager.swift]: Our [Solana Web3 script is now used in Brave-Core (v1.47.152 and later)](https://github.com/brave/brave-browser/issues/27325). When we update core to v1.47, our checked-in `SolanaWeb3Script.js` will be removed and we'll restore behaviour to using the script from core. Updating this script is tracked with https://github.com/brave/brave-ios/issues/6680.

**Review by Brandon-T — APPROVED** [review]

**Pull request merged**